public/js/auth.js

```diff
@@ -48,12 +125,16 @@ function add_user() {
   var par_success = document.getElementById("user-created");
   var par_not_success = document.getElementById("user-not-created");
 
+  var par_error = document.getElementById("cant-remove-user");
+
   overload_xhr(
     "POST",
     "/api/user",
     function(){
       par_success.style.display = "block";
       par_not_success.style.display = "none";
+      par_error.style.display = "none";
+      refresh();
     },
     function(xhr){
       xhr.setRequestHeader('user', user);
```

Comment:
Use double quotes instead of single quotes (yes there are a lot of them in the project)